options/db_options.h

```diff
@@ -116,7 +116,6 @@ struct ImmutableDBOptions {
 struct MutableDBOptions {
   static const char* kName() { return "MutableDBOptions"; }
   MutableDBOptions();
-  explicit MutableDBOptions(const MutableDBOptions& options) = default;
```

Comment:
What is the error related to this copy constructor?

Reply:
Here is the error message:
```
./options/db_options.h:119:12: error: definition of implicit copy assignment operator for 'MutableDBOptions' is deprecated because it has a user-declared copy constructor [-Werror,-Wdeprecated-copy]
  explicit MutableDBOptions(const MutableDBOptions& options) = default;
           ^
db/db_impl/db_impl.cc:1229:27: note: in implicit copy assignment operator for 'rocksdb::MutableDBOptions' first required here
      mutable_db_options_ = new_options;
```